### DISCUSSION TITLE
Make Global Variable available to all Langflow Users

### DISCUSSION THREAD

**Discussion by greenmotion** [langflow-ai/langflow#4048] [General]
Hello,

How can I make a global variable available to all users?

For instance, I configured a PostgresQL Vector Database and stored it's connection URL (with credentials) as a Credential Global Variable. How do I make this Global Variable available to the other Langflow users so that they can connect to the VectorDB without me having to share the connection and/or credentials with them?

Thank you,

  GM

**Comment by severfire** [comment]
Very good idea. Another good idea would be sharing projects with other users :-) with edit lock feature. (only one can edit at once)
